tests/cephadm/test_remove_cluster.py

```diff
@@ -0,0 +1,98 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CephadmOpsFailure(Exception):
+    pass
+
+
+def _disable_mgr_module(node):
+    """Disable cephadm module"""
+    cmd = "cephadm shell -- ceph mgr module disable cephadm"
+
+    result = node.exec_command(sudo=True, cmd=cmd)
+    if result:
+        log.info("Disabled cephadm module sucessfully")
+        return
+
+    raise CephadmOpsFailure("Failed to disable cephadm module")
+
+
+def _get_cluster_fsid(installer):
+    """Get cluster FSID"""
+    cmd = "cephadm shell -- ceph fsid"
+    result = installer.exec_command(sudo=True, cmd=cmd)
+
+    if result:
+        log.info("Cluster FSID: {}".format(result[0]))
+        return result[0].strip()
+
+    raise CephadmOpsFailure("Failed to get cephadm cluster FSID")
+
+
+def _remove_cephadm_cluster(nodes, fsid):
+    """Remove cephadm cluster from nodes"""
+    cmd = "cephadm rm-cluster --fsid {} --zap-osds --force".format(fsid)
+
+    for node in nodes:
```

Comment:
Are you sure this needs to be run on every ceph nodes, when some of the nodes don't have `cephadm` won't it fail there?

Reply:
@sunilangadi2 Yes, it need to be executed on all nodes. As part of `test_cephadm.py` test case `cephadm` is installed on all nodes.